src/help/plugin.ts

```diff
@@ -149,19 +149,21 @@ function activate(app: JupyterLab, mainMenu: IMainMenu, palette: ICommandPalette
   // Handle state restoration.
   restorer.restore(tracker, {
     command,
-    args: widget => ({ isHidden: widget.isHidden, url: widget.url }),
-    name: widget => namespace
+    args: widget => ({ url: widget.url }),
```

Comment:
Now the titles are there upon restoring, but closing the help tabs doesn't remove them from the restored state, so any refresh will bring them back even if you have closed them. Any ideas on how to fix that?